netbox_onboarding/tests/test_netbox_keeper.py

```diff
@@ -308,7 +308,9 @@ def test_ensure_primary_ip_not_exist(self):
         nbk.ensure_device()
 
         self.assertIsInstance(nbk.nb_primary_ip, IPAddress)
-        self.assertEqual(nbk.nb_primary_ip.interface.name, "ge-0/0/0")
+        self.assertTrue(
```

Comment:
FWIW you could also use `self.assertIn()` here if you wanted to.

Reply:
Thanks - makes sense!